### DISCUSSION TITLE
外部测试申请

### DISCUSSION THREAD

**Discussion by imfuxiao** [imfuxiao/Hamster#37] [Q&A]
为了提高App的质量, 希望有空闲时间的朋友可以進行外部测试, 

申请链接: https://testflight.apple.com/join/6AkRfQYZ

谢谢大家.

**Comment by Attiv** [comment]
提示不接受任何成员

**Reply by Attiv** [reply]
@imfuxiao

**Reply by imfuxiao** [reply]
估计是还没有审核通过测试版本造成的 等测试审核通过后 我在看看吧

**Reply by Attiv** [reply]
😂还没过呐，大佬加油，那个librime 编译我还没通过，貌似要单独编译 boost 的filesystem

**Reply by imfuxiao** [reply]
我咋觉得外部测试版本审核比正式版本审核要慢.

对的 不能使用brew安装的boost, librimeKit包下有两个gitsubmodule,一个是boost 一个是librime 两个都写的有编译脚本 应该可以无脑make直接编译 你具体看看makefile里面的步骤

**Reply by Attiv** [reply]
我改成官网下源码编译了，还是报那个错，正在排查问题，/usr/local/include/boost/ 下也有filesystem 那三个库

**Reply by Attiv** [reply]
对了，testflight 外部测试可以用的状态是这样的
![CleanShot 2023-04-15 at 23 30 28@2x](https://user-images.githubusercontent.com/15698129/232234299-03ed2045-7e54-49a7-ad75-f9be2fbfed18.png)
经常会出现需要改下这个的东西：
![CleanShot 2023-04-15 at 23 31 20@2x](https://user-images.githubusercontent.com/15698129/232234353-fc5514a4-326b-4590-a32e-8dd6c50ce5ab.png)

**Reply by imfuxiao** [reply]
你可以参考下脚本 librime编译时需要指定boost的home_path

**Reply by imfuxiao** [reply]
我这边昨天提交的和今天提交的都显示还在审核中

**Reply by Attiv** [reply]
testglight 第一次审核后只要不修改版本号只改build version 的话很快就能过，改了版本号就要重新审核了

**Reply by imfuxiao** [reply]
嗯嗯 跟正式版本一样 睡了 晚安

**Reply by Attiv** [reply]
大佬晚安

**Comment by blycoris** [comment]
中英文切换不好用，切换成英文切不回去，用九宫格英文键盘还是9键，建议可以切成26键英文

**Reply by imfuxiao** [reply]
麻烦把你的问题单独提交一个 issue, 并将问题描述的清晰一些，最好能配上截图或视频，方便我更好的了解您的问题。

**Comment by blycoris** [comment]
已提交issue
